### ISSUE TITLE
Language matches 22% correct

### ISSUE THREAD

**Issue by labrys** [guessit-io/guessit#259]
Running a scrape of rarbg's rss feed, with no filters on types of results, and processing all names with guessit 2.0rc7 and limiting output to only those with a language match: 
##### Total unique release names with language: 27
- de: 6
- fr: 4
  - `ElegantRaw.15.12.30.Zorah.White.Zorah.Gets.A.Lesson.In.French.XXX.1080P.MP4-GUSH`
  - `French Maid Services : Lola At Your Service - Marc Dorcel WEB-DL SPLIT SCENES MP4-RARBG`
  - `French Maid Services - Lola At Your Service- Marc Dorcel Web-DL Split Scenes mp4`
  - `French Maid Services - Lola At Your Service WEB-DL SPLIT SCENES MP4-RARBG`
  - Incorrect matches on French in titles
- av: 5
  - `StreetBlowJobs.16.01.24.Ava.Steel.Feeling.Blue.XXX.1080p.MP4-KTR`
  - `ATKExotics.16.01.24.Ava.Alba.Masturbation.XXX.1080p.MP4-KTR`
  - `ATKExotics.16.01.24.Ava.Alba.Watersports.XXX.1080p.MP4-KTR`
  - `BigTitsRoundAsses.com_15.08.20.Ava.Addams.Big.Tits.Ava.Addams.Gets.An.Anal.Excavation.XXX.iMAGESET-YAPG`
  - `StreetBlowJobs.16.01.24.Ava.Steel.Feeling.Blue.XXX.1080p.MP4-KTR`
  - Incorrect matches on name `Ava`
- es: 3
  - `PutaLocura.15.12.18.Spanish.Glory.Hole.Carlota.Teen.3.XXX.720p.MP4-oRo`
  - `PutaLocura.15.12.22.Spanish.Luzzy.XXX.720p.MP4-oRo`
  - 2 Incorrect matches on `Spanish` in title
- sc: 3
  - `Second.Chance.S01E02.One.More.Notch.720p.WEB-DL.DD5.1.H264-SC[rartv]`
  - `Second.Chance.S01E02.One.More.Notch.1080p.WEB-DL.DD5.1.H264-SC[rartv]`
  - `Total.Divas.S05E01.720p.HDTV.AAC2.0.H.264-SC-SDH`
  - Incorrect matches [H264 Scalable Video Coding](https://en.wikipedia.org/wiki/Scalable_Video_Coding)
- ms: 2
  - `Mr.Selfridge.S04E03.1080p.WEB-DL.AAC2.0.H264-MS[rartv]`
  - `Mr.Selfridge.S04E03.720p.WEB-DL.AAC2.0.H264-MS[rartv]`
  - Incorrect matches [MS-H264](https://msdn.microsoft.com/en-us/library/hh659565%28v=office.12%29.aspx)?
- hy: 1
  - `The.Man.With.The.Golden.Arm.1955.1080p.BluRay.x264.DTS-FGT`
  - Incorrect match on `Arm` in title
- kn: 1
  - `Bjyukujyo Kyoushi Kan XXX 720P WEBRIP MP4-GUSH`
  - Incorrect match on name `Kan`
- pl: 1
  - `Playboy Nederland January 2016-PL`
  - Likely incorrect match to `PL` of indeterminate usage
- ae: 1
  - `Duck.Dynasty.S09E04.Drone.Survivor.720p.AE.WEBRip.AAC2.0.H264-BTW[rartv]`
  - Incorrect match to network `AE` (i.e. `A&E`)

Also country codes are capital, while language codes are lowercase.  Is this intentional?
log: https://gist.github.com/labrys/8c8592a453d3aaab7132

**Comment by Diaoul** [comment]
Case is defined as per their respective ISO standards so yes this is intentional.

**Comment by labrys** [comment]
@Diaoul good to know, and as always appreciate all the work you do, and the quick responses.

**Comment by Toilal** [comment]
It definitively makes me think that guessit should, by default, try to guess most common languages only, instead of trying to ban all exotic languages from the planet, issue after issue ...

**Comment by labrys** [comment]
@Toilal or maybe just fall back to exotic languages only after all other matches have been made

**Comment by duramato** [comment]
@Toilal Other solution could be a blacklist of known problematic languages

**Comment by Toilal** [comment]
@duramato there's already a blacklist of words, but it's hard to maintain and there are always new case incoming that fails because some words are not in the blacklist ...

see https://github.com/guessit-io/guessit/blob/master/guessit/rules/common/words.py#L40-L76

**Comment by labrys** [comment]
@Toilal possibly add support for custom blacklist updates?  That would allow adding blacklist items on a per-case basis that might not be critical for main guessit and as a solution between updates to the blacklist.

**Comment by ratoaq2** [comment]
I'm sharing the list of languages and countries that we're using since long time (and with good results):

```
allowed_languages = {
    'de',
    'en',
    'es',
    'fr',
    'he',
    'hu',
    'it',
    'jp',
    'nl',
    'pl',
    'pt',
    'ro',
    'ru',
    'sv',
    'uk',
}

allowed_countries = {
    'us',
    'gb',
}
```

**Comment by ratoaq2** [comment]
@labrys 
How sure you are about:
```
Second.Chance.S01E02.One.More.Notch.720p.WEB-DL.DD5.1.H264-SC[rartv]
Second.Chance.S01E02.One.More.Notch.1080p.WEB-DL.DD5.1.H264-SC[rartv]
```
and
`Total.Divas.S05E01.720p.HDTV.AAC2.0.H.264-SC-SDH`

Incorrect matches H264 Scalable Video Coding

The first group I'm pretty sure it's a release group: `SC`. It just follow the convention
The second group makes sense. And `SDH` is the release_group in that case.


For the following:
```
Mr.Selfridge.S04E03.1080p.WEB-DL.AAC2.0.H264-MS[rartv]
Mr.Selfridge.S04E03.720p.WEB-DL.AAC2.0.H264-MS[rartv]
```
`MS` is a `release_group`. It just follows the convention as well. Last piece and dash separated.

**Comment by labrys** [comment]
@ratoaq2 They very well could be the release group instead, but they definitely weren't Sardinian and Malaysian languages.

**Comment by ratoaq2** [comment]
Fixed by #476
